pyproject.toml

```diff
@@ -1,10 +1,97 @@
 [build-system]
-build-backend = "setuptools.build_meta"
-requires = [
-    "setuptools>=45.0",
-    "wheel>=0.37.0",
+requires = ["poetry-core>=1.9.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+# Check https://python-poetry.org/docs/pyproject/ for all available sections
+name = "ansys-fluent-core"
+version = "0.20.dev1"
+description = "PyFluent provides Pythonic access to Ansys Fluent"
+license = "MIT"
+authors = ["ANSYS, Inc. <ansys.support@ansys.com>"]
+maintainers = ["PyAnsys developers <pyansys.maintainers@ansys.com>"]
+readme = "README.rst"
+repository = "https://github.com/ansys/pyfluent"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+packages = [
+    { include = "ansys", from = "src" },
+    { include = "tests", to = "ansys/fluent" },
+    { include = "README.rst", to = "ansys/fluent/core/docs" },
+    { include = "datamodel_222", from = "src/ansys/fluent/core", to = "ansys/fluent/core" },
+    { include = "datamodel_231", from = "src/ansys/fluent/core", to = "ansys/fluent/core" },
+    { include = "datamodel_232", from = "src/ansys/fluent/core", to = "ansys/fluent/core" },
+    { include = "datamodel_241", from = "src/ansys/fluent/core", to = "ansys/fluent/core" },
+    { include = "datamodel_242", from = "src/ansys/fluent/core", to = "ansys/fluent/core" },
+    { include = "tui_222", from = "src/ansys/fluent/core/meshing", to = "ansys/fluent/core/meshing" },
+    { include = "tui_231", from = "src/ansys/fluent/core/meshing", to = "ansys/fluent/core/meshing" },
+    { include = "tui_232", from = "src/ansys/fluent/core/meshing", to = "ansys/fluent/core/meshing" },
+    { include = "tui_241", from = "src/ansys/fluent/core/meshing", to = "ansys/fluent/core/meshing" },
+    { include = "tui_242", from = "src/ansys/fluent/core/meshing", to = "ansys/fluent/core/meshing" },
+    { include = "settings_222", from = "src/ansys/fluent/core/solver", to = "ansys/fluent/core/solver" },
+    { include = "settings_231", from = "src/ansys/fluent/core/solver", to = "ansys/fluent/core/solver" },
+    { include = "settings_232", from = "src/ansys/fluent/core/solver", to = "ansys/fluent/core/solver" },
+    { include = "settings_241", from = "src/ansys/fluent/core/solver", to = "ansys/fluent/core/solver" },
+    { include = "settings_242", from = "src/ansys/fluent/core/solver", to = "ansys/fluent/core/solver" },
```

Comment:
We need to include codegen folders because we are installing before and after codegen?

There is also tui_*.py in solver folder and fluent_version_*.py in core folder.

Reply:
Yes, Thank you.

Reply:
@mkundu1 Yes, I verified this. 

We have commented disabled API generated files section in `.gitignore`

![image](https://github.com/ansys/pyfluent/assets/106588300/c5f23dab-3b12-40cc-a4be-ed0b05f8df80)

We even don't need `include = [{ path = "src/ansys/fluent/core/solver/settings_242", format = ["sdist", "wheel"] },]` the generated files are included by poetry.

Thank you very much.

Reply:
@hpohekar We shouldn't modify the `.gitignore` file, instead we should reinstate the explicit includes with globbing patterns.

Reply:
Ok. sure. We will include all of them using globbing patterns. Thank you.

Reply:
Done. Thank you.